modules/4337/contracts/Safe4337Module.sol

```diff
@@ -211,6 +211,45 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
         operationHash = keccak256(operationData);
     }
 
+    /**
+     * @dev Checks if the signatures length is correct and does not contain additional bytes. The function does not
+     * check the integrity of the signature encoding, as this is expected to be checked by the {Safe} implementation
+     * of {checkSignatures}.
+     * @param signatures signatures data
+     * @param threshold Indicates the number of iterations to perform in the loop.
+     * @return isValid True if length check passes, false otherwise.
+     */
+    function _checkSignatureLength(bytes calldata signatures, uint256 threshold) internal pure returns (bool isValid) {
+        uint256 expectedOffset = threshold * 0x41;
+
+        for (uint256 i = 0; i < threshold; i++) {
+            bool pointsAtEnd = true;
+            /* solhint-disable no-inline-assembly */
+            /// @solidity memory-safe-assembly
+            assembly {
+                let signaturePos := mul(0x41, i)
+                // read the Safe signature type byte from the signatures bytes, this is the 64th byte per signature.
```

Comment:
```suggestion
                // read the Safe signature type byte from the signatures bytes, this is the 64th byte per signature.
```


I just realize that my suggestion was ambiguous, it is the 65th byte, or byte at index 64.

Reply:
https://github.com/safe-global/safe-modules/pull/453/commits/025234faf4e2a372a76c4b6c740c220e498513c4